src/metabase/sync/interface.clj

```diff
@@ -9,11 +9,12 @@
 
 (mr/def ::DatabaseMetadataTable
   [:map
-   [:name           ::lib.schema.common/non-blank-string]
-   [:schema         [:maybe ::lib.schema.common/non-blank-string]]
-   [:require-filter {:optional true} :boolean]
+   [:name                                  ::lib.schema.common/non-blank-string]
+   [:schema                                [:maybe ::lib.schema.common/non-blank-string]]
+   [:esetimated-row-count {:optional true} [:maybe :int]]
```

Comment:
>  when the driver supports :sync-estimated-row-count. 

I meant to remove that feature since we don't have a partial reason for having it now. maybe we need to let third party drivers know about it but it doesn't have much benefit in syncing it now, do you it still worth mentioning in driver log?

Reply:
also made this schema a closed map to avoid situations like this.

Reply:
As I said on slack I guess we technically don't need a driver feature yet, but when we use the row count we should definitely add the feature to help with testing and optimizations. So up to you. There's not much of a downside if we introduce a driver feature, never use it, then remove it later, so I might just add one now.

Reply:
> made this schema a closed map

Hmm maybe we want to be careful here, assuming we want to turn on malli schema validation in production again, making this map closed constitutes a breaking change. Driver implementations can currently return whatever keys they want in this map, and if we turn malli schema checking on this will break their sync process. Unlikely, but possible

Reply:
both fair points, I guess we can add the feature now, and also open up the schema again